guides/v2.3/graphql/reference/customer.md

```diff
@@ -117,3 +121,417 @@ The following call returns information about the logged-in customer.
+            is_subscribed
+        }
+    }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "createCustomer": {
+      "customer": {
+        "id": 5,
+        "firstname": "Bob",
+        "lastname": "Loblaw",
+        "email": "bobloblaw@example.com",
+        "is_subscribed": true
+      }
+    }
+  }
+}
+```
+### Update a customer
+
+Updates the customer's personal information.
+
+#### Syntax
+
+`mutation: updateCustomer`
+
+#### Example usage
+
+The following call updates the first and last name and email address for a specific customer.
+
+**Request**
+
+``` text
+mutation {
+    updateCustomer(
+        input: {
+            firstname: "Robert"
+            lastname: "Loblaws"
+            email: "robloblaws@example.com"
+            password: "b0bl0bl@w"
+        }
+    ) {
+        customer {
+            firstname
+            lastname
+            email
+        }
+    }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "updateCustomer": {
+      "customer": {
+        "firstname": "Robert",
+        "lastname": "Loblaws",
+        "email": "robloblaws@example.com"
+      }
+    }
+  }
+}
+```
+
+### Manage customer address {#managecustomeraddress}
+{:.no_toc}
+
+Use these mutations to create or modify the customer's address.
+
+**Manage customer address attibutes**
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`id` | Int | The ID assigned to the address object
+`CustomerAddressInput` | [CustomerAddress](#customerAddress) | An array containing the customer’s shipping and billing addresses
+
+### Create customer address
+
+Creates the customer's address.
+
+#### Syntax
+
+`mutation: createCustomerAddress`
+
+#### Example usage
+
+The following call creates an address for the specified customer.
+
+**Request**
+
+``` text
+mutation {
+  createCustomerAddress(input: {
+    region: {
+        region: "Arizona"
+        region_id: 4
+        region_code: "AZ"
+    }
+    country_id: US
+    street: ["123 Main Street"]
+    telephone: "7777777777"
+    postcode: "77777"
+    city: "Phoenix"
+    firstname: "Bob"
+    lastname: "Loblaw"
+    default_shipping: true
+    default_billing: false
+  }) {
+    id
+    customer_id
+    region {
+      region
+      region_id
+      region_code
+    }
+    country_id
+    street
+    telephone
+    postcode
+    city
+    default_shipping
+    default_billing
+  }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "createCustomerAddress": {
+      "id": 4,
+      "customer_id": 5,
+      "region": {
+        "region": "Arizona",
+        "region_id": 4,
+        "region_code": "AZ"
+      },
+      "country_id": "US",
+      "street": [
+        "123 Main Street"
+      ],
+      "telephone": "7777777777",
+      "postcode": "77777",
+      "city": "Phoenix",
+      "default_shipping": true,
+      "default_billing": false
+    }
+  }
+}
+```
+
+### Update customer address
+
+Updates the customer's address.
+
+#### Syntax
+
+`mutation: updateCustomerAddress`
```

Comment:
Use the same format shown in the Graphiql browser

Reply:
fixed